src/main.zig

```diff
@@ -84,7 +84,7 @@ const normal_usage =
     \\
     \\Commands:
     \\
-    \\  build            Build project from build.zig
+    \\  build            Build project from build.zig with subcommands to run, test etc.
```

Comment:
```suggestion
    \\  build            Build project from build.zig with subcommands
```
If anything I'd make it this (but honestly I'd just not change it) because not every build.zig supports a run or test subcommand.